usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -28,6 +28,17 @@ def preprocess(keyword):
     return keyword
 
 
+def es_sanitizer(keyword):
+    """ Escapes reserved elasticsearch characters and removes when necessary """
+    keyword = concat_if_array(keyword)
+    processed_string = re.sub(r'([-&!|{}()^~*?:\/"])', r'\\\1', keyword)
+    processed_string = re.sub('[<>]', "", keyword)
+    if len(processed_string) != len(keyword):
+        msg = "Stripped characters from ES city search string New: '{}' Original: '{}'"
+        logger.info(msg.format(processed_string, keyword))
```

Comment:
I doubt it, but can anyone think of a PII concern?

Reply:
no